crates/nu-command/Cargo.toml

```diff
@@ -22,6 +22,10 @@ unicode-segmentation = "1.8.0"
 glob = "0.3.0"
 thiserror = "1.0.29"
 sysinfo = "0.20.4"
+bigdecimal = { package = "bigdecimal-rs", version = "0.2.1", features = ["serde"] }
+num-bigint = { version="0.3.1", features=["serde"] }
+num-format = { version="0.4.0", features=["with-num-bigint"] }
+num-traits = "0.2.14"
```

Comment:
Oops, I should have mentioned in the porting guide - engine-q doesn't use bigint or bigdecimal like nushell. We've moved to i64/f64. For porting, if you use the i64/f64 methods, you should be good.

Reply:
Fixed it :)